node/runtime/pangolin/src/pallets/ecdsa_authority.rs

```diff
@@ -17,7 +17,7 @@ static_assertions::const_assert!(SyncInterval::get() < MaxPendingPeriod::get());
 impl Config for Runtime {
 	type ChainId = ChainId;
 	type Event = Event;
-	type MaxAuthorities = MaxAuthorities;
+	type MaxAuthorities = MaxEcdsaAuthorities;
```

Comment:
Suggest renaming `type MaxAuthorities` to `type MaxEcdsaAuthorities`.

Reply:
I think `MaxAuthorities` is better.

Reply:
I see. It is easy to make mistakes with the property in the grandpa pallet, which can lead to configuration errors, especially for novices.  I just have this concern.

Reply:
I don't think so.
It is under the `darwinia_ecdsa_authority::Config`.
It is not easy to make mistakes here.

Just like `type PalletId = XXX`.
No need to specify
```
dvm::Config	{ type DvmPalletId }
staking::Config {type StakingPalletId }
treasury::Config { type TreasuryPalletId }
```